recipe/meta.yaml

```
@@ -1,56 +1,59 @@
-{% set version = "0.9.2" %}
+{% set version = "0.11.1" %}
 
 package:
   name: trio-websocket
   version: {{ version }}
 
 source:
-  - folder: dist
-    url: https://pypi.io/packages/source/t/trio-websocket/trio-websocket-{{ version }}.tar.gz
-    sha256: a3d34de8fac26023eee701ed1e7bf4da9a8326b61a62934ec9e53b64970fd8fe
-  - folder: src
     url: https://github.com/HyperionGray/trio-websocket/archive/refs/tags/{{ version }}.tar.gz
-    sha256: acc72db00e89e4594caac269aabebeccbd7b2d87d4dd06acba7adf4344cd8a42
+    sha256: 31b33561be82ea8f748ca2d8adaf3c45a2003c1a5e1550d7cf44e5a884564bce
 
 build:
   number: 0
-  noarch: python
-  script: cd dist && {{ PYTHON }} -m pip install . -vv
+  skip: True  # [py<37]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed -vvv
 
 requirements:
   host:
     - pip
-    - python >=3.5
+    - python
+    - wheel
   run:
-    - async_generator >=1.10
-    - python >=3.5
+    - python
     - trio >=0.11
     - wsproto >=0.14
+    - exceptiongroup # [py<311]
 
 test:
   source_files:
-    - src/pytest.ini
-    - src/tests
+    - pytest.ini
+    - tests
   imports:
     - trio_websocket
   commands:
     - pip check
-    - cd src && pytest -vv --cov trio_websocket --cov-report term-missing:skip-covered --no-cov-on-fail --cov-fail-under 95
+    - pytest -vv --cov trio_websocket --cov-report term-missing:skip-covered --no-cov-on-fail --cov-fail-under 95 # [not arm64]
```

```suggestion
    - pytest -vv  # [not arm64]
```
We can remove the coverage report as it's not important for us.
Was there a reason to skip tests on osx-arm64? Could you please leave a comment? Is it possible to skip failing tests on arm64?